ding/model/template/bc.py

```diff
@@ -1,10 +1,12 @@
 from typing import Union, Optional, Dict, Callable, List
 import torch
 import torch.nn as nn
+from easydict import EasyDict
 
 from ding.torch_utils import get_lstm
 from ding.utils import MODEL_REGISTRY, SequenceType, squeeze
-from ..common import FCEncoder, ConvEncoder, DiscreteHead, DuelingHead, MultiHead
+from ..common import FCEncoder, ConvEncoder, DiscreteHead, DuelingHead, \
+        MultiHead, RegressionHead, ReparameterizationHead
 
 
 @MODEL_REGISTRY.register('bc')
```

Comment:
rename to discrete bc

Reply:
I merge continous bc and discrete bc into one policy bc. How can I name it?